doc/release/upcoming_changes/21262.improvement.rst

```diff
@@ -0,0 +1,19 @@
+``np.kron`` now supports masked arrays
```

Comment:
No need to add a release note, since this worked before; you are just correcting a bug that nobody will have seen except if they tried the development version.

Reply:
Hey @mhvk , yep, although we did lose the masked array info. But yeah makes sense, it was a bug of sorts. Will remove this :+1: